src/Persistence/Proxy.php

```diff
@@ -26,7 +26,7 @@ interface Proxy
      *
      * Acts as a no-op if already initialized.
      */
-    public function __load(): void;
+    public function __load();
```

Comment:
For this one, I get `PHP Fatal error:  Declaration of Doctrine\Tests\Proxies\__CG__\Doctrine\Tests\Models\Cache\Country::__load(): Doctrine\ORM\Proxy\ProxyFactory must be compatible with Doctrine\Persistence\Proxy::__load(): void in /home/greg/dev/doctrine-orm/patch/vendor/symfony/var-exporter/LazyGhostTrait.php on line 81`.

Cc @nicolas-grekas

Reply:
this could be avoided by changing the ORM ProxyFactory to alias the implement a `__load` function forwarding the call to the `initializeLazyObject` method of the LazyGhostTrait (and importing `initializeLazyObject` as private) instead of renaming the trait method itself (which returns `$this`)

Reply:
and this can be done in ORM 3.x IMO

Reply:
See https://github.com/doctrine/orm/pull/11511